docs/Installation/Installation.md

```diff
@@ -327,77 +282,71 @@ yaml-cpp@develop~shared` (note that is a tilde and not a dash in front
-    `mkdir build && cd build`
-4.  Build SpECTRE with
-    `cmake -D CHARM_ROOT=CHARM_DIR/ARCH_OPTS SPECTRE_ROOT`
-    then
-    `make -jN`
-    to compile the code.
-5.  Run the tests with
-    `make test-executables && ctest`.
-
-**Notes**:
-* For more details on building Charm++, see the directions
-  [here](http://charm.cs.illinois.edu/manuals/html/charm++/A.html)
-  The correct target is `charm++` and, for a personal machine, the
-  correct target architecture is likely to be `multicore-linux-x86_64`
-  (or `multicore-darwin-x86_64` on macOS).
-  On an HPC system, the correct Charm++ target architecture depends on the
-  machine's inter-node communication architecture. We will be providing specific
-  instructions for various HPC systems.
-* Both Charm++ and SpECTRE must be compiled using the same compiler,
+## Building Charm++
+
+If you are not using a container, haven't installed Charm++ with Spack, or want
+to install Charm++ manually for other reasons, follow the installation
+instructions in the [Charm++ repository](https://github.com/UIUC-PPL/charm)
+and in their [documentation](https://charm.readthedocs.io/en/latest/quickstart.html#installing-charm).
+Here are a few notes:
+
+- Once you cloned the [Charm++ repository](https://github.com/UIUC-PPL/charm),
+  run `git checkout v6.10.2` to switch to a supported, stable release of
+  Charm++.
+- Choose the `LIBS` target to compile. This is needed so that we can support the
+  more sophisticated load balancers in SpECTRE executables.
+- On a personal machine the correct target architecture is likely
+  `multicore-linux-x86_64`, or `multicore-darwin-x86_64` on macOS. On an HPC
+  system the correct Charm++ target architecture depends on the machine's
+  inter-node communication architecture.
+- Both Charm++ and SpECTRE must be compiled using the same compiler,
   otherwise you will receive undefined reference errors while linking SpECTRE.
   When compiling Charm++ you can specify the compiler using, for example,
   ```
-  ./build charm++ ARCH clang
-  ```
-  When compiling SpECTRE you can specify the compiler to CMake using,
-  for example,
-  ```
-  cmake -D CMAKE_CXX_COMPILER=clang++ \
-        -D CMAKE_C_COMPILER=clang \
-        -D CMAKE_Fortran_COMPILER=gfortran \
-        -D CHARM_ROOT=CHARM_DIR/ARCH_OPTS SPECTRE_ROOT
+  ./build LIBS ARCH clang
   ```
-* Inside the SpECTRE build directory, use `make list` to see all available
-  targets. This list can be refreshed by running CMake again.
+
+## Building SpECTRE {#building-spectre}
+
+Once you have set up your development environment you can compile SpECTRE.
+Follow these steps:
+
+1. Create a build directory where you would like to compile SpECTRE. In the
+   Docker container you could create, e.g., `/work/spectre-build`. It can be
+   useful to add a timestamp or descriptive label to the name of the build
+   directory since you may create more later, e.g., `build_YYYY-MM-DD` or
```

Comment:
let's not endorse version/date numbering directories... That's literally what version control was created to eliminate. I think if anything, having `build-compiler-mode` like you have is all we should say.